js/bybit.js

```diff
@@ -873,16 +1108,18 @@ module.exports = class bybit extends Exchange {
             //             "deadlineDay":27,
             //             "expiredAt":"",
             //             "createdAt":"",
-            //             "unified":1
+            //             "unified":1,
+            //             "uta": 1
             //         },
             //         "retExtInfo":null,
             //         "time":1669735171649
             //     }
             //
             const result = this.safeValue (response, 'result', {});
             this.options['enableUnifiedMargin'] = this.safeInteger (result, 'unified') === 1;
```

Comment:
@sc0Vu can't we fall into this problem here?
![image](https://user-images.githubusercontent.com/43336371/219692044-eb97aed5-d886-4f1d-bd4d-832ccddd5b55.png)

Reply:
@carlosmiei api go through v5 when user enable unified account, by setting in options or `uta == 1`